src/main/java/ui/PanelMenuCreator.java

```diff
@@ -48,15 +41,33 @@ public Menu generatePanelMenu() {
         items.add(createLeftPanelMenuItem());
         items.add(createRightPanelMenuItem());
         items.add(closePanelMenuItem());
-        autoCreateItems.add(createCustomizedPanelMenuItem(ASSIGNEE_PANEL_NAME, ASSIGNEE_FILTER_NAME));
-        autoCreateItems.add(createCustomizedPanelMenuItem(MILESTONE_PANEL_NAME, MILESTONE_FILTER_NAME));
-        autoCreateItems.add(createCustomizedPanelMenuItem(UPDATED_PANEL_NAME, UPDATED_FILTER_NAME));
+        for (Map.Entry<String, String> entry :
+                generatePanelDetails(lastLoginUsername).entrySet()) {
+            autoCreateItems.add(createPanelMenuItem(entry.getKey(), entry.getValue()));
+        }
         autoCreatePanelMenu.getItems().addAll(autoCreateItems);
         panelMenu.getItems().addAll(items);
         panelMenu.getItems().add(autoCreatePanelMenu);
         return panelMenu;
     }
 
+    /**
+     * Returns a map of custom panels that can be created using the auto-create menu
+     * with the key of the map as the panel name and the value as the corresponding filter name.
+     * Uses the username as a parameter to construct the filter names.
+     * @param lastLoginUsername
+     */
+    public static Map<String, String> generatePanelDetails(String lastLoginUsername){
+        Map<String, String> customPanel = new LinkedHashMap<>();
+        customPanel.put("Open issues and PR's",
+                String.format("is:open ((is:issue assignee:%1$s) OR (is:pr author:%2$s))", lastLoginUsername,
+                        lastLoginUsername));
```

Comment:
I suggested the positional arguments so we don't have to repeat the variable here.